src/TodoList.jsx

```diff
@@ -1,36 +1,35 @@
+  import React from 'react';
+  import TodoListItem1 from './TodoListItem1';
 
-import './index.css'
+  const todoList = [
+    {
+      objectID: 1,
+      title: "Create a repo on GitHub"
+    },
+    {
+      objectID: 2,
+      title: "Create the todo application"
+    },
+    {
+      objectID: 3,
+      title: "Complete assigment"
+    }
+  ];
+  
+  const TodoList = () => {
+    
+    return (
+      <div>
+        <ul>
+        {todoList.map(todo => (
+            <TodoListItem1 key={todo.id} todo={todo} />
```

Comment:
This won't crash your app, but there is no 'id' property on the todo objects. You have the property named 'objectID', so if you have a console open, it may say that there are no keys because an 'id' property doesn't exist